org.lflang/src/org/lflang/generator/c/CPreambleGenerator.java

```diff
@@ -70,6 +76,12 @@ public static String generateDefineDirectives(
         CodeBuilder code = new CodeBuilder();
         code.pr("#define LOG_LEVEL " + logLevel);
         code.pr("#define TARGET_FILES_DIRECTORY " + addDoubleQuotes(srcGenPath.toString()));
+
+        //TODO: Define Global Support for 32-bit and Microsecond Granularity on Reactor-C
```

Comment:
What does this comment refer to? If it is something you're not planning to address in this PR, please create an issue and reference it in the `TODO` comment.